{"name": "test_get_task_negative", "status": "passed", "description": "Проверка получения задачи с некорректным ID", "steps": [{"name": "Получение задачи с фейковым ID 34534534234", "status": "passed", "start": 1748198213891, "stop": 1748198215836}], "start": 1748198213891, "stop": 1748198215837, "uuid": "8bcd4c9e-ac2a-4c8b-9e53-a248db960a72", "historyId": "44de99435f5a5a70aa6a7f419db40385", "testCaseId": "44de99435f5a5a70aa6a7f419db40385", "fullName": "tests.api.api_tests#test_get_task_negative", "labels": [{"name": "feature", "value": "Негативные тесты получения задачи"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "api_tests"}, {"name": "host", "value": "PC-Andrei"}, {"name": "thread", "value": "13352-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.api_tests"}]}